SonarQube.MSBuild.Tasks/GetAnalyzerSettings.cs

```diff
@@ -0,0 +1,83 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="GetAnalyzerSettings.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using SonarQube.Common;
+
+namespace SonarQube.MSBuild.Tasks
+{
+    /// <summary>
+    /// Build task to return the Roslyn analyzer settings from the analysis config file
```

Comment:
Simple build task to read the config file from disc and return the values as properties on the task.